pyriemann/utils/mean.py

```diff
@@ -733,3 +734,144 @@ def nanmean_riemann(covmats, tol=10e-9, maxiter=100, init=None,
         sample_weight=sample_weight
     )
     return C
+
+
+###############################################################################
+
+
+def median_euclid(covmats, tol=10e-6, maxiter=50, init=None):
+    """Median of matrices according to the Euclidean metric.
+
+    Return the Euclidean geometric median for matrices,
+    different from the marginal median provided by numpy.median().
```

Comment:
can you add the formula of what is computed in ..math mode?

Reply:
Great idea! it's done!